crates/api_crud/src/post/create.rs

```diff
@@ -168,12 +181,16 @@ pub async fn create_post(
     .with_lemmy_type(LemmyErrorType::CouldntLikePost)?;
 
   mark_post_as_read(person_id, post_id, &mut context.pool()).await?;
+  send_webmention(inserted_post, community);
```

Comment:
I'm guessing this send_webmention should also be moved into the block above, since it hasn't actually been published and doesn't need to be mentioned yet. 

Also need to add that to the scheduled task.

Reply:
Right, fixed